packages/medusa/src/strategies/tax-calculation.ts

```diff
@@ -101,11 +101,13 @@ class TaxCalculationStrategy implements ITaxCalculationStrategy {
     for (const sm of shipping_methods) {
       const lineRates = taxLines.filter((tl) => tl.shipping_method_id === sm.id)
       for (const lineRate of lineRates) {
-        taxTotal += calculatePriceTaxAmount({
-          price: sm.price,
-          taxRate: lineRate.rate / 100,
-          includesTax: taxInclusiveEnabled && sm.includes_tax,
-        })
+        taxTotal += Math.round(
```

Comment:
Rounding the number inside the loop can lead to a big difference in the total.
This is a private method, and `calculate` is already rounding the total.

Reply:
can I let you do the changes please :)